pi-client/src/main/scala/org/kaloz/pi4j/client/pi/PiGpio.scala

```diff
@@ -0,0 +1,63 @@
+package org.kaloz.pi4j.client.pi
+
+import com.pi4j.wiringpi.{Gpio => JGpio, GpioInterruptCallback}
+import org.kaloz.pi4j.client.Gpio
+
+class PiGpio extends Gpio {
```

Comment:
as I asked. is this needed?
just omit the weaving and you will got the same.
probably it not worth the hassle to maintain..